File: app-java/src/main/java/com/tritiumstudios/phasmophobiaevidencetool/android/activities/investigation/mapsmenu/mapdisplay/data/models/MapDimension.java
package com.tritiumstudios.phasmophobiaevidencetool.android.activities.investigation.mapsmenu.mapdisplay.data.models;

import android.util.Log;

public class MapDimension {

    public final int w;
    public final int h;

    public MapDimension(int w, int h) {
        this.w = w;
        this.h = h;
    }

    public synchronized void print() {
        Log.d("Maps", "[ W: " + w + " H: " + h + " ]");
    }
}

File: app-java/src/main/java/com/TritiumGaming/phasmophobiaevidencepicker/activities/investigation/mapsmenu/mapdisplay/data/models/MapDimension.kt
package com.TritiumGaming.phasmophobiaevidencepicker.activities.investigation.mapsmenu.mapdisplay.data.models

import android.util.Log

class MapDimension(val w: Int, val h: Int) {
    @Synchronized
    fun print() {
        Log.d("Maps", "[ W: $w H: $h ]")
    }
}